### PR TITLE
added updating information to Readme

### PR DESCRIPTION
**Pull request by Jolanrensen**
especially for datalore very useful

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -65,6 +65,65 @@ see [Jupyter docs](https://jupyter-client.readthedocs.io/en/stable/kernels.html#
 
 Uninstall: `./gradlew uninstall`
 
+## Updating
```
Comment (ileasile) [inline docs/README.md:68]: Please run move it to README_STUB.md and run `./gradlew generateReadme`

File: docs/README.md [hunk 2]
```diff
@@ -65,6 +65,65 @@ see [Jupyter docs](https://jupyter-client.readthedocs.io/en/stable/kernels.html#
...
+For example, for the stable version:
+
+`conda install -c jetbrains kotlin-jupyter-kernel={VERSION}` ([package home](https://anaconda.org/jetbrains/kotlin-jupyter-kernel))
+
+### Pip
+
+To update the kernel using Pip, simply run:
+
+Stable:
+`pip update kotlin-jupyter-kernel` ([package home](https://pypi.org/project/kotlin-jupyter-kernel/))
```
Comment (ileasile) [inline docs/README.md:107]: It seems that it works in a bit different way
https://stackoverflow.com/a/47071257/5488904

File: docs/README.md [hunk 3]
```diff
@@ -65,6 +65,65 @@ see [Jupyter docs](https://jupyter-client.readthedocs.io/en/stable/kernels.html#
...
+  - kotlin-jupyter-kernel=={VERSION}
+```
+where `{VERSION}` should be replaced by the latest PyPi version of the Kotlin Jupyter kernel, such as `0.11.0.198`.
+Stop and restart the machine afterwards.
+
+### Conda
+
+If you have `conda` installed, just run the following command to update the stable package version:
+
+`conda update -c jetbrains kotlin-jupyter-kernel` ([package home](https://anaconda.org/jetbrains/kotlin-jupyter-kernel))
```
Comment (ileasile) [inline docs/README.md:90]: Let's remove these `package home` links

Comment (Jolanrensen) [reply docs/README.md:90]: only from the Updating section?

File: docs/README.md (hunk 1 above)
Comment (Jolanrensen) [reply docs/README.md:68]: done

File: docs/README.md (hunk 3 above)
Comment (ileasile) [reply docs/README.md]: Yep)

**Pull request merged**